BraintreePayPal/PayPalOneTouch/Analytics/PPFPTITracker.h

```diff
@@ -33,7 +33,7 @@
 
 
 /// The delegate which actually sends the data
-@property (nonatomic, weak, readwrite) id<PPFPTINetworkAdapterDelegate> networkAdapterDelegate;
+@property (nonatomic, weak, readwrite) _Nullable id<PPFPTINetworkAdapterDelegate> networkAdapterDelegate;
```

Comment:
Yes, fixed. I don't work with Objective-C that much anymore, so I just picked what Xcode suggested. Thanks!